starter_code/models/Movie.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+const movieSchema = new Schema({
```

Comment:
Both stars and showtimes should be  
```
type: Array 
```

If you look in the seed file you can see the type of data you're inputting